src/ORM/Queries/SQLAssignmentRow.php

```diff
@@ -68,7 +68,7 @@ protected function parseAssignment($value)
                     $parameters = [$parameters];
                 }
 
-                // @todo Some input sanitisation checking the key contains the
+                // Some input sanitisation checking the key contains the
                 // correct number of ? placeholders as the number of parameters
```

Comment:
```suggestion
```
Please always remove the full todo comment.

Reply:
Updated